### ISSUE TITLE
Real-world Training Stability Issue

### ISSUE THREAD

**Issue by NigelNelson** [nakamotoo/dsrl_pi0#10]
Hello,

Thank you for publishing this excellent work and sharing the relevant code!
I was excited by this approach so I trained a pi0 model with standard IL on real-world pick and place data that achieves ~40% completion rate on its own. I was hoping that DSRL could then help improve this model's capabilities, but I've been unsuccessful in maximizing the sparse rewards.

I've tried adjusting the entropy coefficient, dimensionality of the MLP, and removing the pi0 hidden state input from the policy to simplify the state space. However, none of these experiments result in much improvement in the stability of training.

I've attached a photo of a wandb run when I replicated the papers approach and mimicked the hyper-parameters from the real-world pi0 experiments.

Any advice would be greatly appreciated!

<img width="3340" height="550" alt="Image" src="https://github.com/user-attachments/assets/db4ef088-a462-4831-ba9b-44c86d23a937" />

**Comment by mw9385** [comment]
Hi, @NigelNelson  did you try to fine-tune the action magnitude? Also, what is the x-axis means for the figures? Are they training episodes or number of gradients updates?

I am running the same experiments following the original paper, and it seems like the action magnitude is playing a very important role. Could share more of your hyperparameters?
